src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -3,5 +3,5 @@
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws RegistrationException;
```

Comment:
We shouldn't declare throws for RuntimeExceptions
```suggestion
    User register(User user);
```